test/units/cli/test_galaxy.py

```diff
@@ -1080,7 +1094,13 @@ def test_parse_requirements_in_older_format_illega(requirements_cli, requirement
 - version: 1.0.0
 '''], indirect=True)
 def test_parse_requirements_without_mandatory_name_key(requirements_cli, requirements_file):
-    expected = "Collections requirement entry should contain the key name."
+    # Used to be "Collections requirement entry should contain the key name."
+    # Should we check that either source or name is provided before using the dep resolver?
```

Comment:
Had a question about this one.

Reply:
Not sure. It's already checked when constructing the Requirement objects.